tests/data/packages/BrokenEmitsUTF8/setup.py

```diff
@@ -10,14 +8,10 @@ class FakeError(Exception):
 
 if sys.argv[1] in ("install", "bdist_wheel"):
     if hasattr(sys.stdout, "buffer"):
+        sys.stdout.buffer.write(b"\nThis package prints out UTF-8 stuff like:\n")
+        sys.stdout.buffer.write("* return type of ‘main’ is not ‘int’\n".encode())
         sys.stdout.buffer.write(
-            "\nThis package prints out UTF-8 stuff like:\n".encode("utf-8")
-        )
-        sys.stdout.buffer.write(
-            "* return type of ‘main’ is not ‘int’\n".encode("utf-8")
-        )
-        sys.stdout.buffer.write(
-            "* Björk Guðmundsdóttir [ˈpjœr̥k ˈkvʏðmʏntsˌtoʊhtɪr]".encode("utf-8")
+            "* Björk Guðmundsdóttir [ˈpjœr̥k ˈkvʏðmʏntsˌtoʊhtɪr]".encode()
```

Comment:
btw, why did one of these lines drop the `encode` and become a binary string, but the others didn't?

Reply:
I assumed because it's pure ASCII so a bytestring is unambiguous? I'm not a huge fan of tools making changes like this (I tend to feel it's a case where I write it a certain way because it expresses what I want it to better that way) but I'm not going to get up tight about it, frankly.

Reply:
The question wasn't about the argument it was about why the first line uses a bytes literal and the next two lines use regular strings + encode.

@pfmoore had the right idea, in fact using non-ASCII characters is illegal for bytes literals: https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals

I do find this ugly though, having 3 lines where they made to be bytes in two different ways. But his isn't my codebase so I'll resolve this line of questioning.